src/Server.elm

```diff
@@ -103,7 +103,7 @@ handleRequest db request =
                 , ( "documentation", Encode.string apiDocUrl )
 
                 -- FIXME: the openapi document should be served by some /openapi API endpoint
-                , ( "openapi", Encode.string "https://wikicarbone.beta.gouv.fr/data/openapi.yaml" )
+                , ( "openapi", Encode.string "/data/openapi.yaml" )
```

Comment:
Note: Eventually we'll expose the openapi document at the root of the API, so this is super temporary anyway.

Reply:
👍 
In any case i'd like to keep that change here as it allows seeing the changes when developping locally (otherwise it keeps the production one in the openapi sandbox).